seml/main.py

```diff
@@ -83,44 +84,83 @@ def main():
-    parser_start.add_argument(
+    parser_start_launch_parent.add_argument(
         '-f', '--filter-dict', type=json.loads,
         help="Dictionary (passed as a string, e.g. '{\"config.dataset\": \"cora_ml\"}') to filter the experiments by.")
-    parser_start.add_argument(
+    parser_start_launch_parent.add_argument(
         '-o', '--output-to-console', action='store_true',
-        help="Print output to console instead of writing it to a log file. Only possible if experiment is run locally.")
-    parser_start.set_defaults(func=start_experiments)
+        help="Print output to console.")
+    parser_start_launch_parent.add_argument(
+        '-nf', '--no-file-output', action='store_true',
+        help="Print output to console.")
+    parser_start_launch_parent.add_argument(
+        '-wg', '--worker-gpus', type=str,
+        help="The IDs of the GPUs used by the local worker. Will be directly passed to CUDA_VISIBLE_DEVICES. Has no "
+             "effect for Slurm jobs.",
+    )
+    parser_start_launch_parent.add_argument(
+        '-wc', '--worker-cpus', type=int,
+        help="The number of CPU cores used by the local worker. Will be directly passed to OMP_NUM_THREADS. Has no "
+             "effect for Slurm jobs.",
+    )
+    parser_start_launch_parent.add_argument(
+        '-we', '--worker-environment-vars', type=json.loads,
+        help="Further environment variables to be set for the local worker. Has no effect for Slurm jobs.",
+    )
+
+    parser_start_launch_parent.add_argument(
+        '-u', '--unobserved', action='store_true',
+        help="Run the experiments without Sacred observers (no changes to the database). "
+             "This also disables output capturing by Sacred, facilitating the use of debuggers (pdb, ipdb).")
+
+    parser_start_launch_parent.add_argument(
+            '-d', '--debug', action='store_true',
+            help="Run a single experiment locally without Sacred observers and with post-mortem debugging. "
+                 "This is equivalent to "
+                 "`--verbose --local --num-exps 1 --set_to_pending --post-mortem --output-to-console`.")
+
+    parser_start_launch_parent.add_argument(
+            '-dr', '--dry-run', action='store_true',
+            help="Only show the associated commands instead of running the experiments.")
+
+    parser_start = subparsers.add_parser(
+            "start",
+            parents=[parser_start_launch_parent],
+            help="Fetch staged experiments from the database and run them (by default via Slurm).")
+    parser_start.add_argument(
+            '-l', '--local', action='store_true',
+            help="Run the experiments locally (not via Slurm).")
+
+    parser_start.add_argument(
+        '-nw', '--no-worker', action='store_true',
+        help="Do not launch a local worker after setting experiments' state to PENDING.")
+
+    parser_start.set_defaults(func=start_experiments, set_to_pending=True)
```

Comment:
`set_to_pending` sets all `STAGED` experiments to `PENDING`. For `start`, we set this to true; for `launch-worker`, however, we set it to false because we only want to process `PENDING` jobs, not add additional jobs to the execution queue. Underscores are only in the `set_defaults` function because keyword arguments cannot have dashes in Python.

Reply:
Ah, so the user can't access these. Okay, that makes sense.